### DISCUSSION TITLE
Character Can't Move But Can Rotate And Use Powers?

### DISCUSSION THREAD

**Discussion by WithinAmnesia** [flareteam/flare-game#922] [Q&A] [answered]
So I was tinkering around with powers and I found that the player character cannot like run around but can rotate and I replaced my entire powers file from my 1.13 to see if that helps nothing, I can play the regular flare campaign but not my mod. I'm kinda stumped as to what could stop the player character moving but can still rotate in place and use powers. I wonder if its a key bind or something I'm not sure. I'm kinda tuckered out today and could use some help or like I could just take a fresh 1.14 install and piece by piece add in my new mod stuff and like slowly inch towards what breaks the game but I'm too drained tonight to do that and it could be super simple lol.

**Comment by dorkster** [comment, marked as answer]
Make sure you set the player's movement speed in engine/stats.txt: https://github.com/flareteam/flare-game/blob/master/mods/empyrean_campaign/engine/stats.txt#L55

**Reply by WithinAmnesia** [reply]
I got Speed=7.5 but its from before the bug so it was working fine before.

**Reply by WithinAmnesia** [reply]
Are there any hidden power id for movement? I changed the power id trying to wipe all abilities and restart with new ones starting from 1 2 3 4 5 6 etc.?

**Reply by WithinAmnesia** [reply]
I got my game working lol. So okay so somehow I have these test rings on that give like negative speed. I'm not sure what happened where a fresh character had the same bug but these test rings might have auto equipped or something or who knows. I took off the test rings and my guy can move around. Very strange quirky problem on my end lol. It is all good now mhmm.
